Add Ability to Disable Replication Status Endpoints in Listener Configuration (#23547)
hashicorp/vault: website/content/docs/configuration/listener/tcp.mdx

  there is no X-Forwarded-For header or it is empty, the client address will be
  used as-is, rather than the client connection rejected.

> - `disable_replication_status_endpoints` `(bool: false)` - If set true, the
>   replication status endpoints will be disabled for requests sent to this
>   listener.

Reviewer: ```suggestion
- `disable_replication_status_endpoints` `(bool: false)` - Disables replication
  status endpoints for the configured listener when set to `true`.
```

Style correction: write in active voice, avoid "this" as a pronoun

Author: Thanks, this reads much better.
